courses/models.py

```diff
@@ -758,7 +803,11 @@ class ProgramCertificate(TimestampedModel, BaseCertificate):
 
     program = models.ForeignKey(Program, null=False, on_delete=models.CASCADE)
     certificate_page_revision = models.ForeignKey(
-        PageRevision, null=True, on_delete=models.CASCADE
+        PageRevision,
+        null=True,
+        blank=True,
```

Comment:
I think these complaints are about a missing migration. I generated the migration locally and it works.